support-frontend/assets/pages/subscriptions-landing/subscriptionsLanding.scss

```diff
@@ -546,6 +548,57 @@
 	}
 }
 
+/****************** guardian weekly ********************/
+
+.subscriptions__guardian-weekly.subscriptions__product--feature
+{
+  color: #000;
+	background-color: #cadbe8;
+
+  .subscriptions__image-container {
+    right: 0;
+    margin-bottom: 0;
+    position: relative;
+  }
+
+  .subscriptions__copy-container {
+    background-color: #cadbe8;
+  }
+
+  .subscriptions-feature-packshot {
+    width: 100%;
+
+
+    @include mq($from: tablet) {
+      position: absolute;
+      bottom: 0;
+    }
+
+    img {
+      width: 100%;
+      vertical-align: bottom;
+    }
+  }
+
+  .component-button--primary.component-button--guardian-weekly {
+    color: #ffffff;
+    background-color: #333333;
+  }
+
+  .component-button--primary.component-button--guardian-weekly:hover,
+  .component-button--primary.component-button--guardian-weekly:focus {
+    background-color: #121212;
+  }
+
+  .component-button--tertiaryFeature.component-button--guardian-weekly,
+  .component-button--tertiaryFeature.component-button--guardian-weekly:hover,
+  .component-button--tertiaryFeature.component-button--guardian-weekly:focus {
+    color: #333333;
+    border: solid 1px #333333;
+    background-color: transparent;
+  }
+}
+
```

Comment:
I could do this as part of a separate cleanup card? Could you spare some time to explain how Emotion can do this and then follow the process in future? I think #cadbe8 is a new colour to source.

Reply:
@i-hardy @paul-daniel-dempsey I applied these override styles in the `.scss` as the components styles are in `.scss` and it felt easier to follow the styles when they were colocated rather than split between css-in-js and `.scss`. Ideally these styles would all be ported to css-in-js.

Reply:
Fair enough!